dts/bindings/dma/renesas,smartbond-dma.yaml

```diff
@@ -19,3 +19,9 @@ properties:
     type: int
     const: 1
     description: Number of block counts supported
```

Comment:
We should add `"#dma-cells"` under properties, shouldn't we?

Reply:
Correct, we need to define it as `const` so there is no way for an overlay to change it, later on.